README.md

```diff
@@ -136,15 +137,18 @@ npx stylelint "**/*.{css,scss}"
 * Implement unit tests using RSpec.
 * Implement integration tests with RSpec & Capybara.
 
-### Deployment	
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-This project has been deployed on Render (free version). You'll have to wait a few minutes if the server is inactive at some point.
+### Deployment
 
-URL: https://rails-budget-app.onrender.com/	
+This application is deployed on Render. Visit the following link...
+https://rails-budget-app-ls62.onrender.com/
```

Comment:
Good job so far. but in the depolyed version I was not able to signup. There is some internal server error. I will request you to check the log in the render dashboard and fix the error. Additionally, after updating the changes, i will ecmdo uto to redeply with the updated version and run your test cases again.